### DISCUSSION TITLE
The forecast_fitted_values function does not exist for my AutoMLForecast object

### DISCUSSION THREAD

**Discussion by egolub22** [Nixtla/mlforecast#364] [Q&A]
I instantiated an AutoMLForecast object with several models and fitted it with data. I then tried to run the "forecast_fitted_values()" function to get a data frame with trained values. However it appears that this object has no such function. The "results_" attribute also does not seem to output the same information shown in the tutorial.

<img width="1586" alt="object_error" src="https://github.com/Nixtla/neuralforecast/assets/43010144/e419f62a-e344-4b5c-b7ec-df16af6368f6">

**Comment by jmoralez** [comment]
Hey. Can you please open the discussions in the corresponding repo? I've had to move two already, this one is in neuralforecast but refers to mlforecast.

About the fitted values, that feature hasn't been released yet, you can install from github if you want to use it now.
